task1/src/tasks/task1.ts

```diff
@@ -0,0 +1,29 @@
+const readStdinStream = process.stdin;
+const writeStdoutStream = process.stdout;
+
+export function reverseStringFromStdin(str: string): string {
```

Comment:
also there is no need to `export` this function. If we don't need this function elsewhere in the project it is better to keep it without `export`

Reply:
It was exported actually for unit test